src/renderer/components/ActionButton.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button, styled } from '@mui/material';
+import colors from '../colors';
+
+const ActionButton = styled(Button)`
```

Comment:
We shouldn't need to define primary and secondary button components. 
Because of the custom theme that we are using globally, if we define the variant prop of a button as 'primary' or 'secondary' according to the MUI docs, then it should automatically already be styled according to our designs.

Hence, we don't actually need to do any other styling.

Reply:
Now realising, theme is currently not working. Please use the method described above for now and I'll fix the theme.

Reply:
I changed the functionality that is executed when the CloseIcon is clicked. It now closes the modal and resets back to the first view.

Reply:
@hhen0005 update on this? currently im using Standard button and using color="primary"/"secondary". You mentioned something about variants? - sorry i forgot